admin_guide/high_availability.adoc

```diff
@@ -75,66 +79,67 @@ The list of command options described here are a subset that are relevant to thi
-
-== Examples
-
-See what the IP failover configuration would look like if it is created:
-
+////
+
+==== Setting up an IP-based Network Service
+The following steps describe how to setup a highly-available IP-based network
+service with IP failover:
+
+. Label the nodes for the service. This step can be optional if you run the
+service on any of the nodes in your Kubernetes cluster and use VIPs that can
+float within those nodes. This process may already exist within a complex
+cluster, in that the nodes may be filtered by any constraints or requirements
+specified (e.g. nodes with SSD drives, or higher cpu/memory/disk requirements,
+etc.).
++
+The following example labels a highly-available cache service that is
+binding/listening on port 9736 as "ha-cache=geo":
++
+====
+----
+$ openshift kube label nodes openshift-minion-{6,3,7,9} "ha-cache=geo"
+----
+====
+
+. Run the geo cache service with two or more replicas. An example configuration
+for running a geo-cache service
+https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/examples/geo-cache.json[is
+provided here].
++
+IMPORTANT: Be sure to replace the `myimages/geo-cache` docker image referenced in the
+file with your intended image. Also, change the number of replicas to the
+desired amount and ensure the label matches the one used in step one.
++
 ****
-`$ osadm ipfailover <options ...> -o json`
+$ `osc create -n <namespace> -f ./examples/geo-cache.json`
 ****
 
-
-Create an IP failover configuration if it does not already exist:
-
-****
-`$ osadm ipfailover --virtual-ips="1.2.3.4-5,6.7.8.9,10.11.12.13-15" --create`
-****
-
-
-Create an IP failover configuration on a selection of nodes labelled "my-ha-service=har-reporter" (on 4 nodes with 7 virtual IPs monitoring a service listening on port 4242.
-
-****
-`$ osadm ipfailover harreporter --selector="my-ha-service=har-reporter" --virtual-ips="10.245.2.42,10.245.2.100-104,10.245.2.142,10.245.2.242" --watch-port=4242 --replicas=7 --create`
-****
+. Finally, configure the VIPs and failover for the nodes labeled with
+"ha-cache=geo" in step one. Ensure the number of replicas match the number of
+nodes and that they satisfy the constraint or label we used in step one. Specify
+the VIP address and the port number that IP failover should monitor for the
+desired instances:
++
+====
+----
+$ osadm ipfailover ha-geo-cache --replicas=4 --selector="ha-cache=geo" --virtual-ips=10.245.2.101-104 --watch-port=9736 --credentials="$OPENSHIFTCONFIG" --create
+----
+====
+////
++
+As an alternative, the following example creates an IP failover configuration on
```

Comment:
@ramr Sorry, I'm not sure if i fully understand this your comment here. I agree that this example is a little out of context, and I've commented it out (as shown by the ////'s). I didn't want to fully erase the paragraph and the example, because this was the sample command you had placed previously in the bottom of the document under the "Examples" heading. I'm not sure a separate example section is the best approach with this doc, as it's best to put the examples as close to the steps as possible. Have I made the appropriate sense of your comment?